openquake/hazardlib/imt.py

```diff
@@ -317,7 +317,7 @@ def ASH():
 
 
 # secondary IMTs
-sec_imts = 'Disp DispProb LiqProb LiqOccur LSE PGDMax LSD PGDGeomMean LsProb'.split()
+sec_imts = 'Disp DispProb LiqProb LiqOccur LqSE LsSE PGDMax LSD PGDGeomMean LsProb'.split()
```

Comment:
If we are using Lq for liquefaction, can we also change LiqProb and LiqOccur to LqProb and LqOccur to be consistent?